flowdb/bin/build/configurate.py

```diff
@@ -69,6 +69,8 @@ def bool_env(var):
 stats_target = int(
     os.getenv("STATS_TARGET", 10000)
 )  # Default to higher than pg default
+max_locks = int(os.getenv("MAX_LOCKS", 365 * 5 * 4 * (1 + 4)))
```

Comment:
For future reference, what's the logic for the default here?

Reply:
https://github.com/Flowminder/FlowKit/issues/5157#issuecomment-1131735184